### DISCUSSION TITLE
How do you make the tags inside the workflow file conditional AND?

### DISCUSSION THREAD

**Discussion by bopotato0223** [projectdiscovery/nuclei#5536] [General]
When we make a workflow file, we can add multiple values on tags such as tags: cve, ssrf
But it appears to be that it has a conditional OR, it will ruin templates that has either cve or ssrf.

Is there a way that we can make it an AND condition?
On the example above, it will only run templates that has tags of cve and ssrf

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
